odoo_conn.php

```diff
@@ -1,36 +1,38 @@
 <?php
-/*
-Plugin Name: Contact 7 to Odoo connector
-Plugin URI: https://www.jackdane.co.uk
-Description: Connect your WordPress Contact 7 Forms to Odoo
-Version: 0.1.2
-Requires PHP: 7.3
-Author: Jack Dane
-Author URI: https://www.jackdane.co.uk
-*/
-
-add_option("odoo_conn_db_version");
-
-require("vendor/autoload.php");
-
-require("encryption.php");
-
-require("odoo_connector/odoo_connector.php");
-
-require("dependency_check.php");
-
-register_activation_hook(__FILE__, "\\odoo_conn\\dependency_check\\odoo_conn_contact_7_plugin_active");
-
-require("activation.php");
-
-register_activation_hook(__FILE__, "odoo_conn_activation_function");
-
-require("deactivation.php");
-
-register_deactivation_hook(__FILE__, "odoo_conn_deactivation_function");
-
-require("loaded.php");
-
-require("admin/main.php");
-
-?>
\ No newline at end of file
+/**
+ * Plugin Name: Contact Form 7 to Odoo connector
+ * Description: When submitting a form through Contact Form 7, the data is extracted and mapped to the designated fields in Odoo.
+ * Author: Jack Dane
+ * Author URI: https://www.jackdane.co.uk
+ * Plugin URI: https://github.com/Jack-Dane/odoo-wp-plugin
+ * Requires PHP: 7.3
+ * Tested up to: 6.5.3
+ * Text Domain: cf7-odoo-sync
+ * Domain Path: /languages
+ * Version: 0.1.2
+ */
+
+// Exit if accessed directly.
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; 
+}
+
+// Add a new option to the database.
+add_option( 'odoo_conn_db_version' );
+
+/**
```

Comment:
Any reason to use a DocBlock rather than a generic comment?

Reply:
A single-line comment is fine.